unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Avatar/AvatarRenderer.cs

```diff
@@ -44,6 +44,7 @@ public enum VisualCue
         private Coroutine loadCoroutine;
         private List<string> wearablesInUse = new List<string>();
         private bool facialFeaturesVisible = true;
+        private bool ssaoEnabled = true;
```

Comment:
Be wary that sometimes `AvatarRenderer` is inside a pooled object. This means we have to ensure the values are true if this object is pooled. This may be a good opportunity to track those booleans in a specialized class that manages those optimizations.

Reply:
Not sure if related to this, but I just confirmed that facial features can appear as hidden in the backpack, I think this may be related to the pooling issue.

Reply:
I think it's 100% related to that. The status of FacialFeatures is set by the `AvatarShape` so it shouldn't affect the AvatarEditor or the own player representation. So it's for sure an issue with the pooling. I thought we solved that by defaulting the flags in the `Cleanup` but it might not be enough, gonna take a look.

Thanks! nice catch!